t/compat.t

```diff
@@ -239,7 +239,7 @@ ok $mb, "Module::Build->new_from_context";
     );
 
     require ExtUtils::Install;
-    skip "Needs ExtUtils::Install 1.32 or later", 2 * @cases
+    skip "Needs ExtUtils::Install 1.32 or later", int(2 * @cases)
```

Comment:
I suspect this change is going to be best left in your own tree, for the simple reason that "cperl adds type checking that doesn't DWIM yet" isn't really an argument for a change to a module targeting compatible perl implementations.

I think a reasonable rule of thumb would be "if it works in pumpkin perl and Lehmann's stableperl but not in cperl, the workaround belongs in cperl, not here"